redfish-core/lib/pcie.hpp

```diff
@@ -456,6 +457,20 @@ inline void requestRoutesSystemPCIeDevice(App& app)
                             }
                         }
 
+                        if (size_t* property = std::get_if<size_t>(
+                                &pcieDevProperties["LanesInUse"]);
+                            property)
+                        {
+                            if (property == nullptr)
+                            {
+                                messages::internalError(asyncResp->res);
+                                return;
+                            }
+                            asyncResp->res
+                                .jsonValue["PCIeInterface"]["LanesInUse"] =
+                                *property;
+                        }
+
```

Comment:
What about LinkSpeed ?

Reply:
I think we will want to map to GenerationInUse
https://github.com/SunnySrivastava1984/phosphor-dbus-interfaces/blob/09073b0d8c5a7d2639def569dc3d3588b099d2b3/yaml/xyz/openbmc_project/Inventory/Item/PCIeDevice.interface.yaml#L11

Reply:
LinkSpeed - I think this work is already done?
In redfish we wanted to map this to PCIeType, which I think is [here](https://github.com/ibm-openbmc/bmcweb/blob/5fbc4926b4de9a21c28b8099db78b5c424449661/redfish-core/lib/pcie.hpp#L495):

                        if (std::string* property = std::get_if<std::string>(
                                &pcieDevProperties["GenerationInUse"]);
                            property)
                        {
                            std::string generation =
                                analysisGeneration(*property);
                            if (!generation.empty())
                            {
                                asyncResp->res
                                    .jsonValue["PCIeInterface"]["PCIeType"] =
                                    generation;
                            }
                        }

Reply:
If you agree Gunnar, I'll remove LInkSpeed from the title. (I have already removed LinkSpeed from the commit description.)

Reply:
Yeah, let's remove from the title

Reply:
Done.